content/gsoc-2024/index.md

```diff
@@ -0,0 +1,110 @@
++++
+title = "Google Summer of Code 2024 Wrap-up"
+date = 2024-11-21
+[taxonomies]
+authors = ["Aaron Allen"]
+categories = ["Summer of Code", "Haskell.org"]
+tags = ["announcement", "gsoc"]
```

Comment:
So sorry to ask another question, but does it make sense to have `Summer of Code` the category **and** in the tag? If you're open to suggestion, `Summer of Code` seems to be like a good tag name, as it can be related to many categories (teams) like haddock, cabal, HLS, etc.

Reply:
Sure, I wasn't totally clear on the categories vs. tags distinction. I think I will just move `Summer of Code` to `tags` since it isn't a team per se.